program Project_Exception;

{$APPTYPE CONSOLE}

{$R *.res}

uses
  System.SysUtils;
function MyDivFun(Num1,Num2:Integer):Integer;
begin
  //抛出异常信息
  if Num2=0 then raise Exception.Create('除数为0')//后面代码不能执行
  else
    Result:=Num1 div Num2;
end;
var x:Integer;
begin
  Readln(x);
  try
    Writeln(MyDivFun(10,x));
  except
    on E: Exception do
      Writeln(E.Message);
  end;
  Readln;
end.
